moveit_ros/planning/moveit_cpp/include/moveit/moveit_cpp/plan_solutions.hpp

```diff
@@ -39,6 +39,7 @@
 
 #include <algorithm>
 #include <moveit/planning_interface/planning_response.h>
+#include <mutex>
```

Comment:
Why is that required now (and not before)?

Reply:
It build before, probably because we're lucky and the required header was included from somewhere else but I think it is nicer to ensure that everything is included.